apps/costcontrol/test/unit/startup_test.js

```diff
@@ -280,12 +279,36 @@ suite('Application Startup Modes Test Suite >', function() {
         assertFTEStarted(applicationMode, done);
```

Comment:
Please, move this line at the end of the function. To keep coding structure and make asserts the last sentences.